code/modules/mob/living/carbon/xenomorph/castes/Predalien.dm

```diff
@@ -107,10 +107,7 @@ Your health meter will not regenerate normally, so kill and die for the hive!</s
 	kills = min(kills + 1, max_kills)
 
 /datum/behavior_delegate/predalien_base/melee_attack_modify_damage(original_damage, mob/living/carbon/A)
-	if(!iscarbonsizehuman(A))
-		return
-	var/mob/living/carbon/human/H = A
-	if(isspeciesyautja(H))
+	if(isyautja(A))
```

Comment:
No need to change this as they do essentially the same thing and are unrelated to the carbon size. isspecies is also more efficient check as it's not typechecking and instead leaves room for special circumstances.

Reply:
As stated in the PR's description, there was a need to change this as just removing the carbon size check was for some reason not sufficient to make the predalien able to slash other xenomorphs. I've rewritten it to use "isspeciesyautja" again by making the original code only trigger if we're sure the target is a human.